core/lib/async-data-loader.ts

```diff
@@ -34,16 +34,18 @@ export const AsyncDataLoader = {
     this.queue.push(action)
   },
   flush: function (actionContext: AsyncDataLoaderActionContext) {
-    if (!actionContext.category) actionContext.category = DEFAULT_ACTION_CATEGORY
-    const actionsToExecute = this.queue.filter(ac => (!ac.category || !actionContext.category) || (ac.category === actionContext.category && (!ac.executedAt))).map(ac => {
+    const notExecutedActions = this.queue.filter(ac => {
```

Comment:
But `isCategoryEmpty` will allow to execute already executed actions. So it should be rather smth like `actionsToExecute`